www/content/docs/home/getting-started.mdx

```diff
@@ -56,3 +57,46 @@ const Artworks = props => {
   )
 }
 ```
+
+In order to use Artsy's fonts, you'll also want to add our webfont file to the
+CSS in the header of your app. Here are a few examples:
+
+### Using [react-head](https://github.com/tizmagik/react-head)
+
+```
+import { Link } from "react-head"
+
+render() {
+    return (
+      <>
+        <Link
+          href="https://webfonts.artsy.net/all-webfonts.css"
+          rel="stylesheet"
+          type="text/css"
+        />
+      </>
+    )
+  }
+```
+
+### Using [Styled Components](https://www.styled-components.com/docs/api#createglobalstyle)
+
+```
+import { createGlobalStyle } from "styled-components"
+
+const GlobalStyles = createGlobalStyle`
+  @import url("https://webfonts.artsy.net/all-webfonts.css");
+`
+```
```

Comment:
I would toggle the position of the `react-head` example with the styled components example since the react-head and react helmet approaches are similar.